app/bundles/EmailBundle/DataFixtures/ORM/fakeemaildata.csv

```diff
@@ -1,3 +1,3 @@
-"name","emailType","subject","template","language","content","plainText","publishUp","publishDown","readCount","sentCount","revision","variantSettings","variantStartDate","variantSentCount","variantReadCount"
-"Kaleidoscope Conf List","list","Join us at the 2014  Kaleidoscope Conference!","blank","en","a:2:{s:4:\"body\";s:113:\"<div>Dear {contactfield=firstname},</div><div>&nbsp;</div><div>Join us at the 2014 Kaleidoscope Conference!</div>\";s:6:\"footer\";s:71:\"<div>{webview_text}</div><div>&nbsp;</div><div>{unsubscribe_text}</div>\";}","Join us at the 2014  Kaleidoscope Conference!",NULL,NULL,0,0,1,"a:0:{}",NULL,0,0
-"Kaleidoscope Conf Campaign","template","Join us at the 2014  Kaleidoscope Conference!","blank","en","a:2:{s:4:\"body\";s:113:\"<div>Dear {contactfield=firstname},</div><div>&nbsp;</div><div>Join us at the 2014 Kaleidoscope Conference!</div>\";s:6:\"footer\";s:71:\"<div>{webview_text}</div><div>&nbsp;</div><div>{unsubscribe_text}</div>\";}","Join us at the 2014  Kaleidoscope Conference!",NULL,NULL,0,0,1,"a:0:{}",NULL,0,0
+"name","emailType","subject","template","language","CustomHtml","plainText","publishUp","publishDown","readCount","sentCount","revision","variantSettings","variantStartDate","variantSentCount","variantReadCount","DynamicContent"
```

Comment:
Shouldn't be `customHtml` with lvoer case on first letter? Same bellow.

Reply:
No, iirc Mautic sticks "set" in front of it to get to a setter method, so CustomHtml -> setCustomHtml.

I'm not sure this PR is need anymore though, if it is I'll see about rebasing it when I find some time.